packages/webpack-cli/src/plugins/dotenv-webpack-plugin.js

```diff
@@ -0,0 +1,172 @@
+import dotenv from "dotenv-defaults";
+
+const interpolate = (env, vars) => {
+  const matches = env.match(/\$([a-zA-Z0-9_]+)|\${([a-zA-Z0-9_]+)}/g) || [];
+  matches.forEach((match) => {
+    env = env.replace(match, interpolate(vars[match.replace(/\$|{|}/g, "")] || "", vars));
+  });
+  return env;
+};
+
+const isMainThreadElectron = (target) => target.startsWith("electron") && target.endsWith("main");
+
+class Dotenv {
+  constructor(config = {}) {
+    this.config = Object.assign(
+      {},
+      {
+        path: "./.env",
```

Comment:
Default value should be other, please read my issues and old PR for this, also we should allow to specify array here